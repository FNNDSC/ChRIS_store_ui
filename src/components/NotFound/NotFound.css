.not-found {
    text-align: center;
}
.not-found img {
    width: 70%;
    height: 70%;
}
