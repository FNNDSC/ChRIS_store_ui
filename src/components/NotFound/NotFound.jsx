import React from 'react';
import broken from '../../assets/img/not-found.svg';
import { notFound } from './NotFound.module.css';

const NotFound = () => (
  <div className={notFound}>
    <img alt="404 - Page not found" src={broken} />
  </div>
);

export default NotFound;
