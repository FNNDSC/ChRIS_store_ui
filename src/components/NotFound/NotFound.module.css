.not-found {
    text-align: center;
    max-width: 50%;
    margin: 5em auto;
}
.not-found img {
    width: 70%;
    height: 70%;
}
