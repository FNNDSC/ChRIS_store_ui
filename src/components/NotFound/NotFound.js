import React from 'react';
import broken from '../../assets/img/not-found.svg';
import './NotFound.css';


const NotFound = () => (
  <div className="not-found">
    <img alt="404 - Page not found" src={broken} />
  </div>
);

export default NotFound;
