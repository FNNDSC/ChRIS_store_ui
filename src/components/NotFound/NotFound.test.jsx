import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render notFound div', () => {
    expect(wrapper
      .find('div.notFound')
      .find('img'))
      .toHaveLength(1);
  });
});
