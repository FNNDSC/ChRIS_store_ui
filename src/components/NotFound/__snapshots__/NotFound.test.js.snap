// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
}
`;
